apax/nodes/__init__.py

```diff
@@ -1,4 +1,5 @@
+from .analysis import ApaxBatchPrediction
 from .md import ApaxJaxMD
 from .model import Apax, ApaxEnsemble
 
-__all__ = ["Apax", "ApaxEnsemble", "ApaxJaxMD"]
+__all__ = ["Apax", "ApaxEnsemble", "ApaxJaxMD", "ApaxBatchPrediction"]
```

Comment:
Shall we split this into Nodes that work out-of-the-box and models that depend on IPSuite.
With a
```python
with contextlib.supress(ImportError)
   from .analysis import ApaxBatchPrediction
__all__.append("ApaxBatchPrediction")
```

This would also allow the BatchSelection until `ipsuite-core` is ready.